web/src/components/layout.css

```diff
@@ -5,8 +5,9 @@
 } */
 body {
   margin: 0;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  font-family: Galano Grotesque;
```

Comment:
This is already declared [here](https://github.com/bckrm/www-gatsby/blob/production/web/src/components/Layout.js#L58). Can you remove this please?

Reply:
@asasmith done